### PR TITLE
Fix - Bypass WordPress Show Avatars settings and set always true

### PR DESCRIPTION
**Pull request by chetansatasiya**
### All Submissions:

* [x] Have you followed the [Pull Requests Guidelines](https://github.com/buddyboss/buddyboss-platform/wiki/Submitting-Pull-Requests#pull-request-guidelines)?
* [x] Does your code follow the [WordPress' Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?

### Changes proposed in this Pull Request:

Fixes #ISSUE_NUMBER .

### How to test the changes in this Pull Request:

1.
2.
3.

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [x] Have you successfully run tests with your changes locally?

### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.

### CODE REVIEW

File: src/bp-core/bp-core-filters.php [hunk 1]
```diff
@@ -1417,3 +1417,20 @@ function buddyboss_platform_plugin_update_notice() {
...
+ * @param $option
+ * @param $default
+ *
+ * @return mixed|void
+ *
+ * @since BuddyBoss 1.4.8
+ */
+function bp_core_set_default_show_avatars( $value, $option, $default ) {
+	$value = true;
+	return apply_filters( 'bp_core_set_default_show_avatars', $value, $option, $default );
```
Comment (mehulkaklotar) [inline src/bp-core/bp-core-filters.php:1434]: @chetansatasiya can you add phodoc for the new filter provided.

Comment (chetansatasiya) [reply src/bp-core/bp-core-filters.php:1441]: @mehulkaklotar added the doc

**Review by mehulkaklotar — CHANGES_REQUESTED** [review]
@chetansatasiya we should not always send true for WP's comment here https://prnt.sc/uln8u6

It will break the WP's settings and customer will come for support from WP that setting doesn't save. We need different approach here. We can just define true `BP_SHOW_AVATARS` always and based on settings saved in WP, we should hide this option https://prnt.sc/ulnoo1 and the function related to it will always give wp's selected option.

Merge dev here also.

**Review by mehulkaklotar — APPROVED** [review]
@chetansatasiya perfect! 🚀 

Tested!

**Pull request merged**